graylog_hook_test.go

```diff
@@ -284,29 +284,20 @@ func TestStackTracer(t *testing.T) {
 	if !ok {
 		t.Error("Stack Trace is not a string")
 	}
-	stacktraceRE := regexp.MustCompile(`^
-(.+)?logrus-graylog-hook(%2ev2)?.TestStackTracer
-	(/|[A-Z]:/).+/logrus-graylog-hook(.v2)?/graylog_hook_test.go:\d+
-testing.tRunner
-	(/|[A-Z]:/).*/testing.go:\d+
-runtime.*
-	(/|[A-Z]:/).*/runtime/.*:\d+$`)
 
-	// Change the regex when running test on devel version since stack trace on devel is unpredictable
-	if strings.Contains(runtime.Version(), "devel") {
-		stacktraceRE = regexp.MustCompile(`^
-(.+)?logrus-graylog-hook(%2ev2)?.TestStackTracer
-	(/|[A-Z]:/).*/errors.go:\d+
-runtime.skipPleaseUseCallersFrames
-	(/|[A-Z]:/).*/asm.s:\d+
+	// Run the test for stack trace only in stable versions
+	if !strings.Contains(runtime.Version(), "devel") {
```

Comment:
@gravis I changed this to run only in stable versions because it was failing on tip again.

It is really time consuming and pointless to try to fix this on tip. Even if you fix a typo you might need to spend hours to try to fix this tip regex. 

I hope you agree with this change.